docs/only-allow-admins-to-create-new-streams-feature.md

```diff
@@ -0,0 +1,12 @@
+# Only allow admins to create new streams
+
+1. Login with an Admin account.
+2. Locate the cog
+(![cog](/static/images/help/cog.png)) on the main navigation bar and click on
+it to relase the dropdown menu.
+![Settings dropdown](/static/images/help/settings-dropdown-menu-administration.png)
```

Comment:
You need to add another blank line as well as indent the image for it to be included in the bullet.